cumulusci/tasks/metadata/tests/test_package.py

```diff
@@ -120,7 +121,7 @@ def test_parse_items__skips_files(self):
                 "Account.object",
                 "Custom__c.object",
             ):
-                with open(filename, "w") as f:
+                with open(filename, "w") as f:  # noqa: F841
```

Comment:
Again, I'd rather you fix the problem it's warning about instead of suppressing the warning.

```suggestion
                with open(filename, "w"):
```

Reply:
This change is added this in a commit before I read this request.